src/components/Table/Table.jsx

```diff
@@ -41,6 +41,13 @@ const getTable = (data, columns, sortState, sortField, sortClick) => (
                 overflow: `${column.field === 'kills' ? 'visible' : null}`,
                 color: column.color,
               };
+              if (row.value === null && column.field !== 'name') {
```

Comment:
I think a safer fix is to apply a filter on the input array that generates the Records table.  This table is used widely and this check may cause random rows not to appear when we use it elsewhere.